include/gnc_constants.hpp

```diff
@@ -30,13 +30,13 @@ namespace constant {
 constexpr static double pi = 3.141592653589793L;
 
 /** Good old pi (float version). */
-constexpr static float pi_f = static_cast<float>(pi);
+constexpr static float pi_f = pi;
```

Comment:
Why were the `static_cast`s removed?

Reply:
And because it's verbose. We all know an assignment to a float will perform the cast anyway